### ISSUE TITLE
An example of sending a Ston.fi swap transaction in Cookbook

### ISSUE THREAD

**Issue by anton-trunov** [tact-lang/tact-docs#149]
Related issue: #146 

Also, it would be nice to have an example of adding liquidity (as suggested by @Gusarich)

**Comment by novusnota** [comment]
Depends on #142

**Comment by crazybee69** [comment]
I implemented this in my code here: https://github.com/crazybee69/TonDcaBot/blob/main/contracts/ton_dca_wallet.tact#L257

I spent a lot of time trying to send messages including structures, but I couldn't make it work. Native messages with CellBuilder it works well.

**Comment by a-bahdanau** [comment]
Я возьмусь

**Comment by a-bahdanau** [comment]
Hello! Version 2 of ston.fi looks like it's in development. Should I make an example using version 2 contracts or stick to version 1?

**Comment by anton-trunov** [comment]
Let's please target v2 as it's around the corner
